themes/default/content/blog/managing-github-with-pulumi/index.md

```diff
@@ -7,6 +7,11 @@ authors: ["guinevere-saenger"]
 tags: ["go", "migration", "packages", "github-actions", "people-ops", "ci-cd"]
 ---
 
+{{% notes type="info" %}}
+Article updated to reflect that `parentTeamId` of the `Team` resource accepts a `string` instead of an `int` since v5.9.1 of the
```

Comment:
```suggestion
We recently updated this article to reflect that `parentTeamId` of the `Team` resource accepts a `string` instead of an `int` since v5.9.1 of the
```